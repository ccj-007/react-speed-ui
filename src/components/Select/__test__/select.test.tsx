import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Select } from '../../index';

describe('test Select component', () => {
  it('should render', () => {
    const { asFragment } = render(<Select />);
    expect(asFragment()).toMatchSnapshot();
  });
});
