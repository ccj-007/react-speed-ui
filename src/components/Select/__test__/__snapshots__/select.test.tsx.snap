// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Select component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-select"
    style="width: 300px; height: 36px;"
  >
    <div
      class="speed-input-warp"
    >
      <input
        class="speed-input"
        data-testid="input"
        placeholder="输入你的默认项"
        readonly=""
        style="padding-right: 30px;"
        value=""
      />
    </div>
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-angle-down fa-1x speed-icon speed-select-icon"
      color="#aaa"
      data-icon="angle-down"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M201.4 374.6c12.5 12.5 32.8 12.5 45.3 0l160-160c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L224 306.7 86.6 169.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l160 160z"
        fill="currentColor"
      />
    </svg>
  </div>
</DocumentFragment>
`;
